packages/transformers/posthtml/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I don't think we need this file. Was this just copied from another transformer? We should probably remove it there too at some point if so.

Reply:
Yes, it was before I knew what I was doing 😂. I will remove it.